workspace/assets/css/core/fct-position.less

```diff
@@ -0,0 +1,4 @@
+#sticky() {
+	position: sticky;
+	position: -webkit-sticky;
```

Comment:
Line 2 & 3 should be switched in order to have the prefixed version first.

Reply:
indeed @baptLesta